cmake/CMakeLists.txt

```diff
@@ -845,6 +845,10 @@ configure_file(onnxruntime_config.h.in ${CMAKE_CURRENT_BINARY_DIR}/onnxruntime_c
 
 if (onnxruntime_USE_CUDA)
   add_definitions(-DUSE_CUDA=1)
+  if(NOT CMAKE_CUDA_COMPILER AND ${CMAKE_LIBRARY_ARCHITECTURE} STREQUAL "aarch64-linux-gnu")
+    # Cmake can't find NVCC in its usual place on Jetson
```

Comment:
Because you didn't add it in PATH?

Reply:
Well, yes :) But a lot of developers, including myself, tend to forget to do that on the new system. I suppose, for clarity we should either remove CMAKE_LIBRARY_ARCHITECTURE test or the whole block altogether. What would you prefer?

Reply:
I usually have multiple CUDA versions installed on my system, so I always manually set the PATH env to switch between the CUDA versions. And, if you don't do that, you'll likely forget to set LD_LIBRARY_PATH too. I think in this case, you can modify your docker file, modify the PATH env to make the build pass.

Reply:
I removed that block in second commit.